src/commands/integration-test-install-app-platform.yaml

```diff
@@ -17,4 +17,4 @@ steps:
         - run:
             name: "architect/integration-test-install-app-platform: Run apptestctl bootstrap"
             command: |
-              apptestctl bootstrap --kubeconfig-path=~/.kube/config
+              apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
```

Comment:
This fails with the error below even thought the file exists.

Reverting to the previous working version. 🤦‍♂️ 


```
#!/bin/bash -eo pipefail
apptestctl bootstrap --kubeconfig-path=~/.kube/config
Error: stat ~/.kube/config: no such file or directory
panic: {"kind":"unknown","annotation":"stat ~/.kube/config: no such file or directory","stack":[{"file":"/home/runner/work/apptestctl/apptestctl/cmd/bootstrap/runner.go","line":99},{"file":"/home/runner/work/apptestctl/apptestctl/cmd/bootstrap/runner.go","line":74},{"file":"/home/runner/work/apptestctl/apptestctl/main.go","line":52}]}
```